api/src/main/java/jakarta/enterprise/concurrent/ManagedExecutorDefinition.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) 2021 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+
+package jakarta.enterprise.concurrent;
+
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/**
+ * <p>Defines a {@link ManagedExecutorService}
+ * to be registered in JNDI by the container
+ * under the JNDI name that is specified in the
+ * {@link #name()} attribute.</p>
+ *
+ * <p>Application components can refer to this JNDI name in the
+ * {@link jakarta.annotation.Resource#lookup() lookup} attribute of a
+ * {@link jakarta.annotation.Resource} annotation,</p>
+ *
+ * <pre>{@literal @}ManagedExecutorDefinition(
+ *     name = "java:module/concurrent/MyExecutor",
+ *     hungTaskThreshold = 120000,
+ *     maxAsync = 5,
+ *     context ={@literal @}ContextServiceDefinition(
+ *               name = "java:module/concurrent/MyExecutorContext",
+ *               propagated = { SECURITY, APPLICATION }))
+ * public class MyServlet extends HttpServlet {
+ *    {@literal @}Resource(lookup = "java:module/concurrent/MyExecutor",
+ *               name = "java:module/concurrent/env/MyExecutorRef")
+ *     ManagedExecutorService myExecutor;
+ * </pre>
+ *
+ * <p>Resource environment references in a deployment descriptor
+ * can similarly specify the <code>lookup-name</code>,</p>
+ *
+ * <pre>
+ * &lt;resource-env-ref&gt;
+ *    &lt;resource-env-ref-name&gt;java:module/env/concurrent/MyExecutorRef&lt;/resource-env-ref-name&gt;
+ *    &lt;resource-env-ref-type&gt;javax.enterprise.concurrent.ManagedExecutorService&lt;/resource-env-ref-type&gt;
```

Comment:
Please update `javax` to `jakarta`

Reply:
This and the other occurrences of it are also corrected under https://github.com/eclipse-ee4j/concurrency-api/pull/141/commits/6b80fec615630e3b737efad33d754688daab6e60